### PR TITLE
fix(std/testing): assertion diff color

### PR DESCRIPTION
**Pull request by g-plane**
Before:
图片

After:
图片

### CODE REVIEW

**Review by nayeemrmn — CHANGES_REQUESTED** [review]
Have you tried putting the `stripColor` around the `Deno.inspect()` call instead?

L24-L30:
```diff
-     ? Deno.inspect(v, {
+     ? stripColor(Deno.inspect(v, {
        depth: Infinity,
        sorted: true,
        trailingComma: true,
        compact: false,
        iterableLimit: Infinity,
-     })
+     }))
```

**Comment by g-plane** [comment]
@nayeemrmn Applied.

File: std/testing/asserts.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ function createColor(diffType: DiffType): (s: string) => string {
     case DiffType.removed:
       return (s: string): string => red(bold(s));
     default:
-      return white;
+      return (s: string): string => white(s);
```
Comment (nayeemrmn) [inline std/testing/asserts.ts:45]: Revert.

**Review by nayeemrmn — APPROVED** [review]
LGTM 👍

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @g-plane

**Pull request merged**